client/routes/Navigation.jsx

```diff
@@ -60,13 +60,13 @@ const Navigation = () => {
                                 </NavLink>
                             </Menu.Item>
                             <Menu.Item>
-                                <NavLink to="/facilitator">Facilitator</NavLink>
+                                <NavLink to="/cohorts">Cohorts Management</NavLink>
                             </Menu.Item>
                             <Menu.Item>
-                                <NavLink to="/researcher">Researcher</NavLink>
+                                <NavLink to="/researcher">Data Export (to be merged with cohorts)</NavLink>
```

Comment:
Please revert this change, Researcher view is a **different** class of data: only the consented data can be shown there, where Facilitator can see all data for the cohort they are running.

Reply:
I think that researchers and facilitators should actually see the same view, just with different scopes of data. Even if someone were a researcher and a facilitator, this should work in one view which when loaded:

* if I am a facilitator I see all the cohorts I have created with all of the associated run-response data for each one
* if I am a researcher I see the same thing for my cohorts plus consent-only data for all other cohorts.

Reply:
I agree with your points here, thanks for clarifying.